de-lib/src/main/java/org/iplantc/de/theme/base/client/apps/widgets/AppLaunchViewDisplayStrings.properties

```
@@ -4,4 +4,5 @@ dontShow = Do not show me this message again.
 hpcAppWaitTimes = <p> This application runs on XSEDE, the U.S. National Supercomputing Network. Analyses run on this system enter a queue that includes many non-CyVerse users. Wait times for your analyses (jobs) to submit and run may be up to several days for large-memory applications. Please do not resubmit your analysis. If the status of your analysis has not changed from Submitted to Running after several days, contact support@cyverse.org. To learn more about HPC apps, see <a href='https://pods.iplantcollaborative.org/wiki/x/MYOdAQ' target='_blank'> Using HPC Apps in the DE </a>.</p> <p>Apply for your own XSEDE allocation to gain access to additional resources at <a href="https://portal.xsede.org/allocation-request-steps" target="_blank"> XRAS - Submit Allocation Request </a>. Access to this powerful system is made available to CyVerse users at no cost through a grant from the National Science Foundation. </p>
 launchPreviewHeader=Preview of {0}
 createQuickLaunchSuccess = {0} quick launch created successfully.
+quickLaunchDeleted= The selected Quick Launch has been deleted. The selected app will be opened with cleared settings.
```

Since it's possible for apps to have some default params set by the app integrator:
```suggestion
quickLaunchDeleted= The selected Quick Launch has been deleted. The selected app will be opened with its default settings.
```

---

Ah good point, I'd forgotten about default params!